### DISCUSSION TITLE
I don't get any answers from the Vault.

### DISCUSSION THREAD

**Discussion by PetarBB** [pashpashpash/vault-ai#46] [Q&A]
I asked him a bunch of questions that are included in the file I uploaded, but I get no answer about it.

[AJ Styles.docx](https://github.com/pashpashpash/vault-ai/files/11320549/AJ.Styles.docx)

![image](https://user-images.githubusercontent.com/130665551/234223817-e0dbcfbd-feaa-4de2-b8ac-e99d1f0e2f0c.png)

**Comment by pashpashpash** [comment]
@PetarBB .docx files are not supported in the current version -- they need to be either PDFs or plain text/rtf documents. I'll be adding docx support soon, but .doc and .pages will remain unsupported.
